content/tutorials/manage-the-rippled-server/installation/install-rippled-on-ubuntu.md

```diff
@@ -0,0 +1,88 @@
+# Install on Ubuntu or Debian Linux
+
+This page describes the recommended instructions for installing the latest stable version of `rippled` on **Ubuntu Linux 16.04 or higher** or **Debian 9 (Stretch)**, using the [apt](https://help.ubuntu.com/lts/serverguide/apt.html) utility. [Updated in: rippled 1.3.0][New in: rippled 1.3.0]
+
+These instructions install a binary that has been compiled by Ripple.
+
+
+## Prerequisites
+
+Before you install `rippled`, you must meet the [System Requirements](system-requirements.html).
+
+
+## Installation Steps
+
+1. Update apt repositories:
```

Comment:
Should it be 
1. Update APT repositories
?

Reply:
Decided less is more here and went with just "Update repositories" instead.